otx/cli/builder/builder.py

```diff
@@ -155,33 +156,37 @@ def build_task_config(
         """
 
         # Create OTX-workspace
-        # Check whether the workspace is existed or not
-        workspace_path.mkdir(exist_ok=False)
+        workspace_path.mkdir(parents=True, exist_ok=False)
 
         # Load & Save Model Template
-        otx_registry = OTXRegistry(otx_root).filter(task_type=task_type)
-        if model_type:
-            template_lst = [temp for temp in otx_registry.templates if temp.name.lower() == model_type.lower()]
-            if len(template_lst) == 0:
-                raise ValueError(
-                    f"[otx build] {model_type} is not a type supported by OTX {task_type}."
-                    f"\n[otx build] Please refer to 'otx find --template --task_type {task_type}'"
-                )
-            template = template_lst[0]
-        else:
-            template = otx_registry.get(DEFAULT_MODEL_TEMPLATE_ID[task_type.upper()])
-        template_dir = Path(template.model_template_path).parent
+        if not template:
+            otx_registry = OTXRegistry(otx_root).filter(task_type=task_type)
+            if model_type:
+                template_lst = [temp for temp in otx_registry.templates if temp.name.lower() == model_type.lower()]
+                if len(template_lst) == 0:
+                    raise ValueError(
+                        f"[otx build] {model_type} is not a type supported by OTX {task_type}."
+                        f"\n[otx build] Please refer to 'otx find --template --task_type {task_type}'"
+                    )
+                template = template_lst[0]
+            else:
+                template = otx_registry.get(DEFAULT_MODEL_TEMPLATE_ID[task_type.upper()])
+
+        if template is None:
```

Comment:
We already checked "if not template". Is it still can be None?
As I see in the code it can't be None

Reply:
Thanks, edited